server.js

```diff
@@ -0,0 +1,31 @@
+import "babel-polyfill";
+import koa from "koa";
+import webpackDevMiddleware from "koa-webpack";
+import webpackDevConfig  from "./config/webpack.config.dev.js";
+import React from 'react';
+import serve from 'koa-static';
+
+import path from 'path';
+
+//import {handleRender} from './render-react-app.jsx';
+
+const app = new koa();
+
+console.log(`running app in ${app.env} mode`);
+
+app.use(serve(`${__dirname}/public`));
```

Comment:
this middleware serves all requested files if they are located in public dir (and subdirs, I think)